fairlearn/adversarial/_adversarial_mitigation.py

```diff
@@ -445,7 +444,7 @@ def fit(self, X, y, *, sensitive_features=None):
             training data.
         """
         X, Y, A = self._validate_input(X, y, sensitive_features, reinitialize=True)
```

Comment:
I think we shoulddn't introduce a new `Y` here, this seems to need to be 

```suggestion
        X, y, A = self._validate_input(X, y, sensitive_features, reinitialize=True)
```

Reply:
I agree about the consistency! Unfortunately it is everywhere a capital `Y` jus in the function signature a small `y`. I capitalised it in the signature. [6fb9423](https://github.com/fairlearn/fairlearn/pull/1398/commits/6fb9423e1e0608ad75afa771ab78964d60346c8f)

Reply:
by convention on the sklearn side we always have `y`, except for historical reasons where it was `Y` before everything became `y`. So I'd go with `y` here and deprecate `Y`.